### PR TITLE
migrate docs

### PR DESCRIPTION
**Pull request by dragazo**
@brollb This has all of the docs brought over and updates build paths. It just fails from not being able to get services started, which I think I got it down to just env var issues.

One weird part was that the old build script was cd-ing into the browser submodule, which we can't assume exists since this is now in the services submodule rather than the root netsblox one. I can't remember the specific reason why I did that cd, but I think it was just some hack to get storage to work (cause all other path accesses appear to be absolute), in which case it can be safely removed if no longer needed.

### CODE REVIEW

File: src/docs/adv/cli.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
+Command Line Interface
+======================
+
+NetsBlox includes a command line tool providing basic management and troubleshooting functionality at ``./bin/netsblox``.
```
Comment (brollb) [inline src/docs/adv/cli.rst:4]: Just noticing this so making a note here: this also needs to be updated to the current CLI.

**Comment by brollb** [comment]
@dragazo - I updated the docs generation for the cloud changes (and moved it out to "docs/" instead of in "src"). The nginx config file will need to be updated on the deployment to host the docs as static files but otherwise, it should be basically good to go.

Can you take care of the remaining warnings that pop up when building the docs? We might also want to test the docs in the CI, too...

**Comment by dragazo** [comment]
@brollb Ok, I just fixed the warnings I ran into. They were just minor things like dev services that were renamed, or links that used only one underscore (rst anon links need two for some reason).

**Comment by brollb** [comment]
Thanks, @dragazo !

**Pull request merged**